### PR TITLE
The worker can now run from the frontend as well

### PR DESCRIPTION
**Pull request by annando**
This is an experimental functionality. It can act as a replacement if the used system doesn't allow shell scripts that are called by the crontab. This is not tested intensively and will need some more care to be used on production machines.

### CODE REVIEW

**Comment by rabuzarus** [comment]
Nice work. Just one note: Is there a reason why you mix the use of `Cache::get()` and `get_config()`?

File: mod/worker.php [hunk 1]
```diff
@@ -0,0 +1,38 @@
+<?php
+require_once("include/poller.php");
```
Comment (rabuzarus) [inline mod/worker.php]:
```
/**
 * @file mod/worker.php
 * @brief Module for running the poller as frontend process
 */

**Comment by annando** [comment]
@rabuzarus Concerning the different config calls: the dark force of copy&paste is strong inside of me :)

**Comment by annando** [comment]
I just added a small script so that the poller can now be run from a daemon as well. This means that we don't need a crontab entry anymore - just a shell with php.

File: util/daemon.php [hunk 2]
```diff
@@ -0,0 +1,96 @@
+<?php
+/*
+Run the poller from a daemon.
```
Comment (rabuzarus) [inline util/daemon.php]:
```
/**
 * @file util/daemon.php
 * @brief Run the poller from a daemon.
 *
 * This script was taken from http://php.net/manual/en/function.pcntl-fork.php
 */

Comment (annando) [inline util/daemon.php]: Done

**Pull request merged**